### ISSUE TITLE
Crash report: `Error: Addresses should not be equal`

### ISSUE THREAD

**Issue by teamxnomics** [Uniswap/uniswap-interface#1557]
**Bug Description**
  
App crashed

**Steps to Reproduce**

1. Go to ...
2. Click on ...
   ...

**URL**

https://app.uniswap.org/#/add/ETH/0xdac17f958d2ee523a2206206994597c13d831ec7

**Error**

```
Error: Addresses should not be equal
```

**Stacktrace**

```
Error: Addresses should not be equal
    at e.value (https://app.uniswap.org/static/js/main.8d689671.chunk.js:1:70344)
    at Ay (https://app.uniswap.org/static/js/main.8d689671.chunk.js:1:324622)
    at no (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1481097)
    at zo (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1490664)
    at Vs (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1534546)
    at Au (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1520729)
    at Su (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1520657)
    at Tu (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1520520)
    at bu (https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1517486)
    at https://app.uniswap.org/static/js/4.5121d2e8.chunk.js:2:1466865
```

**Device data**

```json5
{
  "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36",
  "browser": {
    "name": "Chrome",
    "version": "90.0.4430.93",
    "major": "90"
  },
  "engine": {
    "name": "Blink",
    "version": "90.0.4430.93"
  },
  "os": {
    "name": "Windows",
    "version": "10"
  },
  "device": {},
  "cpu": {
    "architecture": "amd64"
  }
}
```

**Comment by moodysalem** [comment]
This one is pretty hard to reproduce, we'll wait for another occurrence with a state dump. Probably has something to do with the bug where we would show tokens that were already on the current list in the inactive currency list

**Comment by jeremieflrnt** [comment]
Hi @moodysalem, I was able to reproduce this one every time like this: 
I was on a add liquidity page (https://app.uniswap.org/#/add/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2/0xeb953eda0dc65e3246f43dc8fa13f35623bdd5ed/10000)
and the I change the select pair (WETH) to another token (ETH in this case) and then I got the page "Something went wrong".
It is not possible to add liquidity with ETH, so I guess that's the problem. But it's not obvious like this!
Hope this helps :)
